src/App.js

```diff
@@ -1,25 +1,60 @@
 import React from 'react';
-
+import { Clock } from './components/Clock';
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: 0,
+  }
+
+  render() {
+    const { isClockVisible, clockName } = this.state;
+
+    return (
+      <div className="App">
+        <h1>React clock</h1>
+        <div className="clock">
+          {isClockVisible ? <Clock name={clockName} /> : null}
+        </div>
+        <div className="button">
+          <button
+            className="btn"
+            type="button"
+            onClick={() => {
+              this.setState({ isClockVisible: true });
+            }}
+          >
+            Show Clock
+          </button>
+
+          <button
+            className="btn"
+            type="button"
+            onClick={() => {
+              this.setState({ isClockVisible: false });
+            }}
+          >
+            Hide Clock
+          </button>
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
+          <button
+            className="btn"
+            type="button"
+            onClick={() => {
+              const newName = Math.floor(Math.random() * 100);
 
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+              this.setState({ clockName: newName });
+              // eslint-disable-next-line
```

Comment:
```suggestion
              // eslint-disable-next-line no-console
```
указывай правило которое отключаешь для следующей строки